hooks/bundled.md

```diff
@@ -1,5 +1,7 @@
 # Built-in hooks
 
+| The built-in hooks are being deprecated in favor of [Common Hooks](./common_hooks.md). Please start using Common Hooks.
```

Comment:
maybe we should highlight this like a ProTip or make it bold so it stands out more.